### PR TITLE
Add support for multiple instances

### PR DESCRIPTION
**Pull request by ccremer**
* Adds support for multiple instances. Each instance should go into its own namespace.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] ~Update the ./CHANGELOG.md.~ to be replaced with changelog generator later

### CODE REVIEW

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 1]
```diff
@@ -26,6 +28,7 @@ type:: string
 default:: `syn-thanos`
 
 The namespace in which to create the Grafana dashboard ConfigMaps.
+Multiple instances should't share the same dashboard namespace to avoid name collisions.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:31]: This restriction essentially prohibits you from having Grafana dashboards belonging to different Thanos instances in the same Grafana as far as I understand the implementation.

**Review by simu — COMMENTED** [review]
I'm not sure this is the right approach to make this component multi-instance aware, as there's no real reason why multiple instances of Thanos components can't share a namespace (I'm aware that it will be more work to make the component work for deploying multiple instances in the same namespace).

File: docs/modules/ROOT/pages/references/parameters.adoc (hunk 1 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/references/parameters.adoc]: Right. I've added a commit to address this.

**Review by simu — APPROVED** [review]
LGTM

**Pull request merged**